services/web-server/src/monitor.js

```diff
@@ -46,4 +46,21 @@ monitorManager.register({
   },
 });
 
+monitorManager.register({
+  name: 'requestReceived',
+  title: 'Request Received',
+  type: 'request-received',
+  version: 1,
+  level: 'info',
+  description: 'A GraphQL request has been received.',
+  fields: {
+    operationName: `
+      The name of the graphql query performed. If the operation is anonymous
+      (i.e., the operation is query { ... } instead of query NamedQuery { ... })
+      , then operationName is null.`,
+    variables: 'Object that contains all of the variables passed to the GraphQL query.',
```

Comment:
This might contain secret data for a few queries -- notably login, and also getting secrets.  I'm not sure if it makes sense to try to denylist those specific variables (and risk missing some) or try to write a (huge!) allowlist, or just not log variables at all.  In the REST API, we have a [cleanPayload](https://github.com/taskcluster/taskcluster/tree/master/libraries/api#declaring-methods) function that we use to clean payloads for calls that contain secrets.  Maybe we could do something vaguely similar here, and then only define that function for login and secret?

Reply:
Oh, you're absolutely right. I forgot to think about this. I think we can just avoid logging the variables at all. Forgetting to update this list in the future would put us in a bad position.